calico/felix/futils.py

```diff
@@ -541,6 +541,13 @@ def ipv6_supported():
         return False, ("ip6tables not installed; Calico IPv6 support requires "
                        "Linux kernel v3.3 or above and ip6tables v1.4.14 or "
                        "above.")
+
+    # Check for the existence of the IPv6 NAT table.
+    try:
+        check_call(["ip6tables-save", "--table", "nat"])
```

Comment:
Please add a UT for this case.  Should be a matter of `m_check_call.side_effect = iter([None, FailedSystemCall(...)])`